src/cljs/rems/administration/catalogue_items.cljs

```diff
@@ -78,7 +98,9 @@
  (fn [[catalogue language] _]
    (map (fn [item]
           {:key (:id item)
-           :name {:value (get-localized-title item language)}
+           :name {:value (get-localized-title item language)
+                  :sort-value [(get-localized-title item language)
+                               (- (time-coerce/to-long (:start item)))]} ; secondary sort by created, reverse
```

Comment:
This secondary sort doesn't work when sorting by resource, form or workflow.

Reply:
Secondary support was not built to the table and this is only to improve the functionality of the default sort in this (arguably common) case. I'm not going to implement secondary sort to the table here.